src/test/java/org/jnode/fs/FileSystemTestUtils.java

```diff
@@ -84,4 +84,79 @@ public static FileSystemService createFSService(String className)
 
         return bytes;
     }
+
+    /**
+     * Create a file .
+     *
+     * @param fileName Name of the file  to be created.
+     * @return void.
+     */
+    public static File createFile(String fileName) throws IOException {
```

Comment:
These functions for creating a file/folder will eventually no longer be needed. Please remove.

Reply:
The functions were removed. Thanks